### PR TITLE
docs: zikpin tracer

### PR DESCRIPTION
**Pull request by zirain**
fix: https://github.com/envoyproxy/gateway/issues/3685

### CODE REVIEW

File: site/content/en/latest/tasks/observability/proxy-metric.md
Comment (shawnh2) [inline site/content/en/latest/tasks/observability/proxy-metric.md:1]: the link `https://gateway.envoyproxy.io/latest/tasks/observability/proxy-observability/#metrics` in https://gateway.envoyproxy.io/latest/tasks/observability/gateway-exported-metrics/ also need a update

Comment (zirain) [reply site/content/en/latest/tasks/observability/proxy-metric.md:1]: where's it?

Comment (shawnh2) [reply site/content/en/latest/tasks/observability/proxy-metric.md:1]: image

Comment (shawnh2) [reply site/content/en/latest/tasks/observability/proxy-metric.md:1]: and lets rename it to `EnvoyProxy Metrics`

File: site/content/en/boilerplates/o11y_prerequisites.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 ---
-Follow the steps from the [Quickstart](../quickstart) to install Envoy Gateway and the example manifest.
+Follow the steps from the [Quickstart](/latest/tasks/quickstart) to install Envoy Gateway and the example manifest.
```
Comment (zirain) [inline site/content/en/boilerplates/o11y_prerequisites.md:3]: this's a known issue.

**Review by shawnh2 — APPROVED** [review]

File: site/content/en/boilerplates/o11y_prerequisites.md (hunk 1 above)
Comment (zirain) [reply site/content/en/boilerplates/o11y_prerequisites.md]: fixed

**Review by Xunzhuo — APPROVED** [review]

**Pull request merged**